.chloggen/support_config_file.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: controller
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Support a yaml config file to set all the configuration options exposed by controllers
```

Comment:
Users will need to add a new volume or something with the file or just add it to their container,  right? Maybe we can document how to use this mechanism

Reply:
I mean, you're speedrunning to where I am going - give us a chance to slowly add to the featureset and then make it possible to configure the manager this way. Possibly a helm chart down the road.